lib/graphql/lang/ast/type_info.ex

```diff
@@ -48,22 +44,18 @@ defmodule GraphQL.Lang.AST.TypeInfo do
     Stack.peek(type_info.field_def_stack)
   end
 
+  # FIXME: pattern match on function heads
   def find_field_def(schema, parent_type, field_node) do
     name = String.to_atom(field_node.name.value)
     cond do
-      name == Introspection.meta(:schema)[:name] && schema.query == parent_type ->
+      name == String.to_atom(Introspection.meta(:schema)[:name]) && schema.query == parent_type ->
```

Comment:
instead of converting to an atom every time, you can use `&String.to_existing_atom/1`, or since this is hard coded to strings in `Introspection.meta/1` it's possible it would be possible to add it there as an atom.